### ISSUE TITLE
Upgrade Ubuntu on server

### ISSUE THREAD

**Issue by mlandauer** [openaustralia/publicwhip#1327]
We're currently on 16.04 which went out of "standard" support in April last year. So, it's not even getting security updates.

If we upgrade to 18.04 LTS we will remain in support until April of next year.

Because we're out of support, root SSL certs aren't even up to date so we're even running into problems running our Ansible configuration scripts which is what's really motivating this change now.

**Comment by mlandauer** [comment]
To upgrade it turns out that upgrading elasticsearch (keeping the old version around) would be a pain. It would just be easier to switch over to using the same clustered elasticsearch setup that we're using for PlanningAlerts. I've added a separate issue for this at #1328

**Comment by mlandauer** [comment]
Yesterday I made complete balls-up of upgrading the TVFY server to Ubuntu 18.04 LTS (which is not the latest but is in support until April of next year). First I updated the ansible scripts to get a working TVFY server under 18.04. Then I made a snapshot of the root disk just in case. Then I tried an in-place upgrade to 18.04 (which is the same hacky thing I did in upgrading righttoknow.org.au and oaf.org.au). This went completely wrong and left the server in a half-upgraded state where packages were broken and it was generally very confused. So restored the snapshot onto a new volume and replaced the volume on the instance for TVFY to bring it back. We ended up with about 15 minutes of downtime at the end of yesterday.

So, today I did the more methodical thing. Created a new server based on an Ubuntu 18.04 LTS AMI. Ran the new ansible scripts against it. Copied across the entire `/etc/letsencrypt` directory from the old machine to the new machine (as suggested in https://community.letsencrypt.org/t/move-to-another-server/77985/6). Ran the ansible scripts again to make sure that everything ran to completion. Then, deployed the app in the normal way. Then tested the setup by pointing my local `/etc/hosts` at the new address using the normal name. Once I was happy that was working, switched over DNS and cleaned up the mess I created.
